x-pack/plugins/monitoring/server/lib/metrics/apm/metrics.js

```diff
@@ -284,6 +284,18 @@ export const metrics = {
       defaultMessage: 'HTTP Requests rejected during server shutdown'
     })
   }),
+  apm_responses_errors_internal: new ApmEventsRateClusterMetric({
```

Comment:
@graphaelli The best way to update the test to pass is by following the instructions here:

https://jestjs.io/docs/en/snapshot-testing#updating-snapshots

This is just a Jest snapshot failure, which is because you added a new metric, `apm_responses_errors_internal` and it's not represented in the associated tests.